Makefile

```diff
@@ -1,6 +1,11 @@
 COMMONENVVAR=GOOS=linux GOARCH=amd64
 BUILDENVVAR=CGO_ENABLED=0
 
+RUNTIME ?= podman
```

Comment:
Can we put in docker as default container runtime considering its the most widely known? Also installation section in the README could be updated `Update the container runtime, repoowner,image and imagename in the Makefile` or something more generic